src/components/track-changes/suggestion-list/SuggestionSnippet.tsx

```diff
@@ -7,61 +7,201 @@
+ * All portions of the code written by Atypon Systems LLC are Copyright (c) 2024 Atypon Systems LLC. All Rights Reserved.
  */
-
 import {
   CHANGE_OPERATION,
   CHANGE_STATUS,
   ChangeSet,
   TrackedChange,
 } from '@manuscripts/track-changes-plugin'
-import React from 'react'
+import { schema } from '@manuscripts/transform'
+import parse from 'html-react-parser'
+import React, { useEffect, useState } from 'react'
 import styled from 'styled-components'
 
-export const SuggestionSnippet: React.FC<{
-  suggestion: TrackedChange
-}> = ({ suggestion }) => {
+import {
+  getFootnoteText,
+  getInlineFootnoteContent,
+} from '../../../lib/footnotes'
+import { changeOperationAlias } from '../../../lib/tracking'
+import { getParentNode } from '../../../lib/utils'
+import { useStore } from '../../../store'
+
+interface SnippetData {
+  operation: string
+  nodeName: string
+  content: string
+}
+
+export const SuggestionSnippet: React.FC<{ suggestion: TrackedChange }> = ({
+  suggestion,
+}) => {
+  const [{ doc, view }] = useStore((store) => ({
+    view: store.view,
+    doc: store.doc,
+  }))
+  const [snippet, setSnippet] = useState<SnippetData | null>(null)
+  const [message, setMessage] = useState('')
   const { dataTracked } = suggestion
 
-  const changeTitle = (c: TrackedChange) => {
-    if (ChangeSet.isTextChange(c)) {
-      return c.text
-    } else if (ChangeSet.isNodeChange(c)) {
-      return `${c.nodeType.charAt(0).toUpperCase()}${c.nodeType.slice(1)} ${
-        c.dataTracked.operation
-      }`
-    } else if (ChangeSet.isNodeAttrChange(c)) {
-      return `${c.nodeType.charAt(0).toUpperCase()}${c.nodeType.slice(1)} ${
-        c.dataTracked.operation
-      }`
+  useEffect(() => {
+    const getSnippetData = (): {
+      snippet: SnippetData | null
+      message: string
+    } => {
+      if (ChangeSet.isTextChange(suggestion)) {
+        const parentNode = getParentNode(view.state, suggestion.from)
+        if (parentNode?.type === schema.nodes.footnote) {
+          return {
+            snippet: {
+              operation: changeOperationAlias(dataTracked.operation),
+              nodeName: parentNode.type.name || suggestion.nodeType.name,
+              content: suggestion.text,
+            },
+            message: '',
+          }
+        } else {
+          return { snippet: null, message: suggestion.text }
+        }
+      }
+
+      if (ChangeSet.isNodeChange(suggestion)) {
+        if (suggestion.node.type === schema.nodes.inline_footnote) {
+          return {
+            snippet: {
+              operation: changeOperationAlias(dataTracked.operation),
+              nodeName: suggestion.node.type.spec.name,
+              content: getInlineFootnoteContent(doc, suggestion.attrs),
+            },
+            message: '',
+          }
+        } else if (suggestion.node.type === schema.nodes.footnote) {
+          console.log('suggestion.node', suggestion.node, suggestion)
```

Comment:
@fwaisi, Did you leave this intentionally?

Reply:
No, good catch.

Will remove it shortly